docs/source/tutorial/manage-access-control-list.rst

```diff
@@ -157,9 +157,28 @@ The general process of changing access to a Resource is as follows:
 Set Public Access
 -----------------
 
-.. note::
+Given a Resource's ACL obtained as specified in
+:ref:`manage-acl-change-access-to-resource`, you can:
+
+- Use :doc:`setPublicResourceAccess
+  </api/modules/_acl_class_>` to grant Access Modes
+  to everybody for the Resource itself:
```

Comment:
Q: Do you think we should explicitly mention that everybody refers to whether people are logged in or not like "everybody (whether logged in or not)" Technically, everybody should cover it, but then ... it's hard to tell how people actually read everybody.

Reply:
Yes, that's a good idea - the standard currently also supports targeting just people logged in. (But they're leaning towards removing it, which is why we don't support it - but if we ever add it, this clarification will be helpful, so I'll add it.)